scripts/update-goldens.sh

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
```

Comment:
move scripts from scripts/ to root directory

Reply:
done